### DISCUSSION TITLE
Lazer | Song Select scrolling and grouping

### DISCUSSION THREAD

**Discussion by sourhdhr** [ppy/osu#27024] [Ideas] [answered]
On stable, you used to be able to group maps via a "My Maps". This, as well as grouping altogether seems to be missing. Is there any intention to re-implement this feature?

Also, on stable song select, you are able to right click anywhere off to the side of the beatmap menu which will "jump" up or down in your songs list to where you right clicked. This appears to be missing in Lazer. If this can't be re-implemented, can we put the scroll bar on the right _into the foreground_, and the _osu logo in the bottom right into the background_, so that the scroll bar can be selected even when at the bottom of your song list? Currently it is blocked by the osu logo.

**Comment by Flamiii** [comment, marked as answer]
Grouping in song select is planned (https://github.com/ppy/osu/issues/4919). You can right click in song select to jump through your beatmaps, but you have to enable the setting "Right mouse drag to absolute scroll" first.

**Reply by sourhdhr** [reply]
Thank you! I had seen the absolute scroll setting but it wasn't intuitive enough for me to realize that was its functionality. Appreciate it!
